Libplanet.Explorer/GraphTypes/ActionType.cs

```diff
@@ -13,13 +17,40 @@ public ActionType()
                 "Arguments",
                 resolve: ctx =>
                 {
+                    object ConvertValue(IValue value)
+                    {
+                        Console.WriteLine(value.GetType());
+                        switch (value)
+                        {
+                            case Text text:
+                            case Integer integer:
+                            case Bencodex.Types.Boolean boolean:
+                            case Null n:
+                                return value;
+                            case Binary binary:
+                                return binary.ToString();
+                            case List list:
+                                return list.Select(x => ConvertValue(x));
+                            case Dictionary dictionary:
+                                var d = new Dictionary<object, object>();
+                                foreach (var pair in dictionary)
+                                {
+                                    d.Add(ConvertValue(pair.Key), ConvertValue(pair.Value));
+                                }
+
+                                return d;
+                            default:
+                                throw new Exception("WRONG");
```

Comment:
Can we have a proper exception and message?

Reply:
Oops sorry, it's mistake. I will adjust immediately. :pray:

Reply:
Can you see this again? I'm not sure that it's right to use `ArgumentException` in this case.